packages/@okta/vuepress-site/docs/guides/submit-app/submission1-general/index.md

```diff
@@ -24,7 +26,7 @@ In the General Settings page, you need to fill in the basic information about yo
 
 * **App description** &mdash; Give a general description of your application and what the Okta integration does. For examples, see the overview section for any of the integrations listed on the [OIN](https://www.okta.com/integrations/).
 
-* **App icon** &mdash; Upload a .png, .jpg, or .gif file of your company logo to accompany the integration in the catalog. The image should be less than 100k in size, and have dimensions less than 104 x 37 pixels to prevent resizing.
+* **App icon** &mdash; Upload a .png, .jpg, or .gif file of a logo to accompany your integration in the catalog. The image must have dimensions of less than 1400 by 400 pixels. It also must be 100 kilobytes or smaller in size.
```

Comment:
The max size of the image has been changed to 1Mb.

Reply:
Changed.